templates/textinput.jsx

```diff
@@ -26,71 +27,82 @@ export default function TextInput (props) {
+              ])}
+              key={_index}>
+              {prefix &&
+                <div className="textinput-item__prefix-container">
+                  <label
+                    className="textinput-item__prefix"
+                    id={`${_id}-${index}-aria`}
+                    htmlFor={`${_id}-${index}`}
+                    aria-label={prefix}
+                    dangerouslySetInnerHTML={{ __html: compile(prefix, props) }}
+                  >
+                  </label>
                 </div>
-                <div className="textinput-item__icon textinput-item__incorrect-icon" aria-label={_globals._accessibility._ariaLabels.incorrect}>
-                  <div className="icon" aria-hidden="true" />
+              }
+
+              <div className="textinput-item__textbox-container">
+                <input
+                  className="textinput-item__textbox js-textinput-textbox"
+                  type="text"
+                  placeholder={placeholder}
+                  data-id={`${input}-${index}`}
+                  id={`${_id}-${index}`}
+                  aria-labelledby={prefix && `${_id}-${index}-aria`}
+                  aria-label={placeholder}
+                  defaultValue={userAnswer}
+                  disabled={!_isEnabled}
+                />
+                <div className="textinput-item__state">
+                  <div className="textinput-item__icon textinput-item__correct-icon" aria-label={_globals._accessibility._ariaLabels.correct}>
+                    <div className="icon" aria-hidden="true"/>
+                  </div>
+                  <div className="textinput-item__icon textinput-item__incorrect-icon" aria-label={_globals._accessibility._ariaLabels.incorrect}>
+                    <div className="icon" aria-hidden="true" />
+                  </div>
                 </div>
               </div>
-            </div>
 
-            {suffix &&
-              <div className="textinput-item__suffix-container">
-                <label
-                  className="textinput-item__suffix"
-                  id={`${_id}-${index}-aria`}
-                  htmlFor={`${_id}-${index}`}
-                  aria-label={suffix}
-                  dangerouslySetInnerHTML={{ __html: compile(suffix, props) }}
-                >
-                </label>
-              </div>
+              {suffix &&
+                <div className="textinput-item__suffix-container">
+                  <label
+                    className="textinput-item__suffix"
+                    id={`${_id}-${index}-aria`}
+                    htmlFor={`${_id}-${index}`}
+                    aria-label={suffix}
+                    dangerouslySetInnerHTML={{ __html: compile(suffix, props) }}
+                  >
+                  </label>
+                </div>
+              }
+
+            </div>
+            {_isInteractionComplete && _canShowCorrectness &&
+            <div
+              key={_index}
```

Comment:
As there are two divs in the `<></>` block, they need different `key` values. These should be pre or postfixed with a differentiating string. 
```
key={`a${_index}`}
key={`b${_index}`}
```

@kirsty-hames